aries_vcx/README.md

```diff
@@ -1,62 +1,43 @@
-# Aries-vcx
-Crate implementing Hyperledger Aries protocols and building blocks for building Aries agents for
-both mobile (typically in role of holder, prover) and server use-cases (issuer and verifier).
+# aries-vcx
+`aries-vcx` is more of a "library" rather than "framework". We strive to be not too 
+opinionated and simply provide building blocks for whatever you want to build. 
 
-# Getting started
-Aries-vcx is library, not a framework. We strive to be not too opinionated and simply provide building block for whatever
-you want to build. 
+You can use `aries-vcx` to build both end-user products (mobile wallets in role of credential 
+holder and prover) or server (typically in role of issuer, verifier). 
 
-Generally, the crate allows you to 
+`aries-vcx` provides support to:
```

Comment:
"support for" + ing form (and also better to not start sentence with `aries-vcx` due to capitalization), but I think the original sentence was fine.

Reply:
reverted